cmd/kubeadm/app/cmd/join.go

```diff
@@ -210,8 +210,7 @@ func NewCmdJoin(out io.Writer, joinOptions *joinOptions) *cobra.Command {
 	AddJoinOtherFlags(cmd.Flags(), &joinOptions.cfgPath, &joinOptions.ignorePreflightErrors, &joinOptions.controlPlane, &joinOptions.token)
 
 	// initialize the workflow runner with the list of phases
-	// TODO: append phases here like so:
```

Comment:
IMO we should remove the comment after add all phases.

Reply:
:thinking: i dunno, i mean is kind intuitive that a phase should be added after other ones.
But i can add it, i have not strong preference on it :smile: